3.3/.sti/bin/run

```diff
@@ -1,25 +1,112 @@
 #!/bin/bash
 
-function is_gunicorn_installed() {
-    pip show gunicorn
+# This script uses these environment variables:
+#
+# APP_MODULE: Python dotted path to your WSGI application.
+#             If not provided, tries to find the Python path to a 'wsgi.py' file
+#             in the source tree.
+#             That file is present in Django projects by default.
+# APP_CONFIG: Optional configuration file for gunicorn.
+# APP_FILE:   Optional path to Python script to run your application.
+#             Defaults to 'app.py' if it exists.
+#
+# DISABLE_MIGRATE: if not empty, inhibits execution of 'manage.py migrate'.
+
+BIND_ADDR="0.0.0.0:8080"
```

Comment:
then I would move it down to place where you call those. having it on top makes it look as something we set and allow to customize (or we expect to change that at some point, which is not valid)